symphony/lib/core/class.errorhandler.php

```diff
@@ -87,20 +104,19 @@ public static function isValidThrowable($e)
      *  This function works with both Exception and Throwable
      *  Supporting both PHP 5.6 and 7 forces use to not qualify the $e parameter
      *
+     * @since Symphony 3.0.0
+     *  The method is final
+     *
      * @param Throwable $e
      *  The Throwable object
      * @return string
      *  The result of the Throwable's render function
      */
-    public static function handler($e)
+    final public static function handler($e)
```

Comment:
Curious on why `final`? What problem does this avoid?

Reply:
I am paranoid and do not want to override it by accident.